### ISSUE TITLE
Te hipoly models from steam arent loaded on ironwail

### ISSUE THREAD

**Issue by Sol1vaN** [andrei-drexler/ironwail#349]
Hi there,

I’ve never played the Quake remaster because the Kex engine causes a lot of stuttering, even with 144fps unlocked. This is due to the game logic (gamehz) being capped at 60fps.

Now, with Ironwail, I’m hopeful I can enjoy the remaster with flawless smoothness. Ironwail loads the remaster game, but the guns, monsters, and sounds still remain in their vanilla versions.

I simply copied the entire 'rerelease' folder (Ironwail doesn't detect my Steam game automatically as it’s supposed to). Ironwail can't launch if I use the entire 'rerelease' folder:
![image](https://github.com/user-attachments/assets/30d09b41-eb47-4510-9b86-0f6ddacb9396)
![image](https://github.com/user-attachments/assets/d4458b7e-ffcf-46b7-804f-c070be0a41ec)

So, I moved the 'id1' folder to the working directory, and Ironwail ran:
![image](https://github.com/user-attachments/assets/cdf82aff-5861-4c17-b52e-a6113c925671)
![e1m3_2024-10-20_23-28-08](https://github.com/user-attachments/assets/83aadad3-389d-4c2e-adc0-748e393cf544)

However, the models and sounds remain vanilla.
Only graphical features like dynamic lighting and colored lighting are loaded.
What am I doing wrong?

**Comment by NightFright2k19** [comment]
I don't think you can run it like that. Rename the "rerelease" folder to "id1" (assuming it has pak0.pak in it) and that's it. Provided MD5 models are turned on in the ingame menu, they should be used.

**Comment by Sol1vaN** [comment]
i mean, 'rerelease' folder is there to show that i've copied all the structure, and then moved the 'pak0.pak' file from 'id1' subfolder to 'id1' ironwail folder.

![image](https://github.com/user-attachments/assets/c67c2b48-b851-4779-aece-f4d1fba70012)

If you do this, you'll notice the hipoly models and hq sfx sounds aren't loaded, only the old ones.
For other side, if you do the same on vkquake, the hipoly are loaded without problem.
However, in vkquake the hq sfx sounds still aren't loaded too.

**Comment by Sol1vaN** [comment]
I've tested 0.8.0 and now the models are loaded without problems.
In both ways:
Ugly way: copying only the pak0.pak and creating a new id1 subfolder
Elegant way: copying all the remaster folder and renaming it to id1

About the hq sounds, i've discovered a cvar called "sndspeed" i've changed it to 44100 and the sounds are better now.
The nightdive quake seems do the same, there's no "hq" sounds after all.

But: it would be nice ironwail can load HQ sound bulb (pk3).
Now, Ironwail is very near to be perfect.
And many thanks for all your work, you are the best guys! 🤗

**Comment by markanini** [comment]
The issue is that `sndspeed 44100` is that it mixes any extra lower quality sounds(mod added sfx) using crude interpolation math that clashes sonically with the high quality sounds. This is done for feature parity with legacy executables, but this style of sound mixing should be made optional in 2025.
